docs/README.md

```diff
@@ -65,13 +65,14 @@ For example:
 },
 ```
 
-Every block in this app only has three props in common:
+Every block in this app only has four props in common:
 
 | Prop name          | Type      |  Description | Default value |
 | --------------------| ----------|--------------|---------------|
 | `markers`           |`[string]` | IDs of your choosing to identify the block's rendered message and customize it using the admin's Site Editor. Learn how to use them accessing the documentation on [Using the Markers prop to customize a block's message](https://vtex.io/docs/recipes/style/using-the-markers-prop-to-customize-a-blocks-message). Notice the following: a block's message can also be customized in the Store Theme source code using the `message` prop. |`[]`|
 |  `blockClass`  |  `string`  |  Block  ID  of your choosing to  be  used  in [CSS  customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property).  |  `undefined`  |
 |  `message`  |  `string`  |  Defines the block's default text i.e. the block message. You can also define which text message a block will render on the UI using the admin's Site Editor.  |  `undefined`  |
+|  `multiplyBySelectedQuantity`  |  `boolean`  |  Displays the price multiplied by selected quantity value eg `vtex.quantity-selector`. ⚠️ Warning: this will not look up in to Promotions and Taxes module. |  `false`  |
```

Comment:
```suggestion
|  `multiplyBySelectedQuantity`  |  `boolean`  |  Whether the price should be displayed according to the product quantity chosen by the user (`true`) or not (`false`). Caution: this prop is not integrated with the Promotions and Taxes module, meaning it does not fetch data from it to display the prices. |  `false`  |
```

Reply:
Just confirming it: the `multiplyBySelectedQuantity` prop is really applicable to all of the app's blocks, right?

Reply:
@guerreirobeatriz Thanks for the the heads-up, I'm going to double-check everything ASAP.